packages/tools/examples/ultrasoundAnnotation/index.ts

```diff
@@ -0,0 +1,277 @@
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addButtonToToolbar,
+} from '../../../../utils/demo/helpers';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+import WindowLevelTool from '../../src/tools/WindowLevelTool';
+
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  StackScrollTool,
+  ZoomTool,
+  UltrasoundAnnotationTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { ViewportType } = Enums;
+const { MouseBindings } = csToolsEnums;
+
+const toolGroupId = 'STACK_TOOL_GROUP_ID';
+const leftClickTools = [UltrasoundAnnotationTool.toolName];
+const defaultLeftClickTool = leftClickTools[0];
+const currentLeftClickTool = leftClickTools[0];
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Ultrasound annotations',
+  'Showcases how the bline and pleura annotations of ultrasound images are displayed and rendered correctly'
+);
+
+const content = document.getElementById('content');
+const element = document.createElement('div');
+
+// Disable right click context menu so we can have right click tools
+element.oncontextmenu = (e) => e.preventDefault();
+
+element.id = 'cornerstone-element';
+element.style.width = '500px';
+element.style.height = '500px';
+
+content.appendChild(element);
+
+const instructions = document.createElement('p');
+instructions.innerText =
+  'Middle Click: Window Level\nRight Click: Zoom\n Mouse Wheel: Stack Scroll';
+
+content.append(instructions);
+// ============================= //
+let renderingEngine;
+const viewportId = 'US_STACK';
+
+addButtonToToolbar({
+  onClick: () => {
+    const viewport = renderingEngine.getViewport(viewportId);
+    viewport.scroll(1);
+  },
+  title: 'Scroll',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const viewport = renderingEngine.getViewport(viewportId);
+    viewport.scroll(1);
+  },
+  title: 'Scroll',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    usAnnotation.setActiveAnnotationType(
+      UltrasoundAnnotationTool.USAnnotationType.BLINE
+    );
+  },
+  title: 'Add B-Line annotation',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    usAnnotation.setActiveAnnotationType(
+      UltrasoundAnnotationTool.USAnnotationType.PLEURA
+    );
+  },
+  title: 'Add Pleura annotation',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    usAnnotation.deleteLastPleuraAnnotation();
```

Comment:
Hmmm, i guess since we have setActiveAnnotationType, this one should be deleteAnnotationType(plural)